includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -292,7 +325,7 @@ public function sanitize() {
 	 * @param array      $rule_spec Rule spec.
 	 * @return array Augmented rule spec.
 	 */
-	private function get_rule_spec_list_to_validate( $node, $rule_spec ) {
+	private function get_rule_spec_list_to_validate( DOMElement $node, array $rule_spec ) {
```

Comment:
Again, would suggest not typehinting against actual arrays.
```suggestion
	private function get_rule_spec_list_to_validate( DOMElement $node, $rule_spec ) {
```